gosushi_client/src/components/menuDrawer.scss

```diff
@@ -0,0 +1,66 @@
+.modal-dialog {
+  position: absolute;
+  left: 0;
+  width: 20vh;
+  height: 90%;
+}
+
+.modal-content {
+  height: 100%;
+  width: 100%;
+  background-color: #D1AB83;
```

Comment:
Can we use css variables for colours similar to how we used them in FoodMood?

Reply:
Oops I intended for all the above comments to be part of the review as well (the comments I attached images to are the only ones I really care about)

Reply:
I added 4 colors that were used in the menuDrawer: tan, camel, light-tan and jet.